src/mono/mono/mini/aot-compiler.c

```diff
@@ -9409,6 +9409,30 @@ add_referenced_patch (MonoAotCompile *acfg, MonoJumpInfo *patch_info, int depth)
 	}
 }
 
+static gboolean
+mono_interp_jit_call_can_be_supported (MonoMethod *method, MonoMethodSignature *sig)
+{
```

Comment:
This should be in transform.c, mono_interp_jit_call_supported () could call it.
It can be called by adding a new callback to ee.h.

Reply:
Updated.